tools/importers/SynchronizeTypographyTokens.ts

```diff
@@ -0,0 +1,191 @@
+const axios = require('axios');
+const tokensFilesConfig = require('./configs/SynchronizeTypographyTokensConfig.json');
+import { arrayToMixinFile, arrayToFile, jsonToFile } from './helpers/fileWriter';
+import { recursiveTokensReader } from './helpers/tokensReader';
+import {
+	typographyPrefix,
+	tokensKey,
+	excludedKeys,
+	importVariables,
+	jsonAttributeToCssProperty,
+	resultCssAdditionalLines,
+	resultScssAdditionalLines,
+	tokensTypographyInputAttributes,
+} from './helpers/typographyVariables';
+import {
+	cssFileFirstLine,
+	mixinNameLine,
+	tokenAsCssPropertyLine,
+	closeBracketLine,
+} from './helpers/modifiers';
+import * as dotenv from 'dotenv';
+import {
+	Dict,
+	ITokenJsonObject,
+	IResultJsonObject,
+	ITypographyToken,
+	TypographyConfigFileBin,
+	TypographyBinFiles,
+	JsonAttribute,
+	JsonAttributeValues,
+} from './helpers/structures';
+
+dotenv.config();
+
+const ImportTypographyVariables = (binFilesConfig: TypographyBinFiles, jsonTypography: Object) => {
+	let resultCss: Array<string> = [];
+	let resultScss: Array<string> = [];
+	let resultJsonCss: Dict<Array<IResultJsonObject>> = {};
+
+	resultCss.push(cssFileFirstLine(false, undefined));
+
+	for (let key in jsonTypography) {
+		if (!excludedKeys.includes(key)) {
+			const propertyOptions: JsonAttribute<JsonAttributeValues> = jsonTypography[key];
+
+			for (let variableKey in propertyOptions) {
+				const propertyName: string =
+					typographyPrefix + jsonAttributeToCssProperty[key] + '-' + variableKey;
+				let propertyValue: string | undefined;
+
+				switch (key) {
+					case tokensTypographyInputAttributes.fontSize:
+					case tokensTypographyInputAttributes.lineHeight:
+						const valueBase: number = parseInt(
+							propertyOptions.base.value.replace(/[^0-9]/g, ''),
+						);
+						const valueSplit: Array<string> =
+							propertyOptions[variableKey].value.split('*');
+						const valueMultiplier: number = +valueSplit[1];
+						propertyValue = valueMultiplier
+							? valueBase * valueMultiplier + 'px'
+							: valueBase + 'px';
+						break;
+					case tokensTypographyInputAttributes.letterSpacing:
+						const temporaryValueAsString: string = propertyOptions[
+							variableKey
+						].value.replace(/[^0-9/.]/g, '');
+						const temporaryValueAsNumber: number = +temporaryValueAsString * 0.01;
+						propertyValue = temporaryValueAsNumber + '';
```

Comment:
https://peps.python.org/pep-0020/#:~:text=Explicit%20is%20better%20than%20implicit.
> Explicit is better than implicit.

```suggestion
						propertyValue = temporaryValueAsNumber.toString();
```

Reply:
Fixed.